src/static/hyperhtml/examples/example/react/not-primitive-props/hyper.js

```diff
@@ -0,0 +1,26 @@
+customElements.define(
+  'h-welcome',
+  class HyperWelcome extends HTMLElement {
+    constructor(...args) {
+      super(...args);
+      this.html = hyperHTML.bind(this);
+    }
+
+    get user() { return this._user; }
+    set user(value) {
+      this._user = value;
+      this.render();
+    }
+
+    connectedCallback() { this.render(); }
+    render() {
+      return this.html`<h1>Hello, ${this._user.name}</h1>`;
+    }
+  }
+);
+
+hyperHTML.bind(document.getElementById('root'))`
```

Comment:
specially for one-off operation, you can use `hyperHTML` right away without `bind`

Reply:
Nice to know, but it's better to keep it consistent across examples to avoid confusing readers